src/components/block/FooterCTACard/FooterCTACard.module.css

```diff
@@ -0,0 +1,28 @@
+.body {
+  color: var(--color-white);
+  font: var(--font-body-medium);
+  line-height: 1.3;
```

Comment:
```suggestion
```

`line-height` is already being set with the `--font-body-medium` shortcut that we have, so there's no need to set it to the exact same value again here.